snmp/datadog_checks/snmp/metrics.py

```diff
@@ -59,4 +59,8 @@ def as_metric_with_forced_type(value, forced_type):
     if forced_type == 'monotonic_count_and_rate':
         return {'type': 'monotonic_count_and_rate', 'value': int(value)}
 
+    if forced_type == 'flag_stream':
+        index = int(options['placement']) - 1
+        return {'type': 'gauge', 'value': int(value[index])}
```

Comment:
1. Should we handling IndexError?

1. Should we handle error case when the value is not an int ?